config/disabled.yml

```diff
@@ -90,6 +90,10 @@ Style/SymbolArray:
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#percent-i'
   Enabled: false
 
+Style/UnneededInterpolation:
+  Description: 'Checks for strings that are just an interpolated expression.'
+  Enabled: false
```

Comment:
I didn't want to assume anything. I did this in a followup commit because it required fixing violations in the Rubocop code.